src/Uno.UI.Adapter.Microsoft.Extensions.Logging/Uno.UI.Adapter.Microsoft.Extensions.Logging.csproj

```diff
@@ -1,18 +1,54 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras" ToolsVersion="15.0">
-	<PropertyGroup Condition="'$(MSBuildRuntimeType)'!='Core'">
-		<TargetFrameworks>xamarinmac20;xamarinios10;MonoAndroid12.0;net461;netstandard2.0</TargetFrameworks>
-		<TargetFrameworksCI>MonoAndroid13.0;MonoAndroid12.0;xamarinios10;net461;netstandard2.0;xamarinmac20</TargetFrameworksCI>
+	<PropertyGroup>
+		<TargetFrameworks>netstandard2.0</TargetFrameworks>
+		<TargetFrameworksCI>netstandard2.0</TargetFrameworksCI>
 	</PropertyGroup>
```

Comment:
This used to be required for documentation to be built properly, mainly because of uno.sourcegeneration. As the build passes now, we are good to go. Same for the android binding below.